eng/pipelines/libraries/build-test-job.yml

```diff
@@ -38,6 +39,9 @@ jobs:
         - ${{ format('libraries_build_{0}{1}_{2}_{3}', parameters.osGroup, parameters.osSubgroup, parameters.archType, parameters.buildConfig) }}
       - ${{ if in(parameters.framework, 'allConfigurations', 'net472') }}:
         - ${{ format('libraries_build_{0}_{1}{2}_{3}_{4}', parameters.framework, parameters.osGroup, parameters.osSubgroup, parameters.archType, parameters.buildConfig) }}
+      # Libraries Test also depends on Product, now that the libraries build only depends on corelib
+      - ${{ if ne(parameters.liveRuntimeBuildConfig, '') }}:
+        - ${{ format('{0}_product_build_{1}{2}_{3}_{4}', parameters.runtimeFlavor, parameters.osGroup, parameters.osSubgroup, parameters.archType, parameters.liveRuntimeBuildConfig) }}
```

Comment:
I believe `allConfigurations` shouldn't depend on the whole coreclr build. It should only need corelib as the tests we run there don't use `testhost`.

Reply:
Disregard, `allConfigurations` doesn't use this template at all. We can actually remove the `in(parameters.framework` conditions above and just leave it as:
```yml
- ${{ format('libraries_build_{0}{1}_{2}_{3}', parameters.osGroup, parameters.osSubgroup, parameters.archType, parameters.buildConfig) }}
# Libraries Test also depends on Product, now that the libraries build only depends on corelib
- ${{ if ne(parameters.liveRuntimeBuildConfig, '') }}:
   - ${{ format('{0}_product_build_{1}{2}_{3}_{4}', parameters.runtimeFlavor, parameters.osGroup, parameters.osSubgroup, parameters.archType, parameters.liveRuntimeBuildConfig) }}
```

Reply:
ok will do as a subsequent PR.

Reply:
these are now included in this PR: https://github.com/dotnet/runtime/pull/34460